tests/Feature/ResourceListUpdateBadgeTest.php

```
@@ -0,0 +1,48 @@
+use App\Models\Project;
+use App\Models\Server;
+use App\Models\Service;
+use App\Models\StandaloneDocker;
+use App\Models\Team;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Cache;
+
+uses(RefreshDatabase::class);
+
+beforeEach(function () {
+    InstanceSettings::forceCreate(['id' => 0]);
```

PHPStan (2.2.8)

[warning] Call to an undefined static method App\Models\InstanceSettings::forceCreate().


(staticMethod.notFound)